owners/test/owner.test.js

```diff
@@ -79,14 +111,32 @@ describe('owner teams', () => {
 
   describe('toString', () => {
     it("returns the team members' usernames as a comma-separated list", () => {
-      expect(owner.toString()).toEqual('auser, anothermember');
+      expect(owner.toString()).toEqual('[auser, anothermember]');
+    });
+
+    it('includes any modifiers', () => {
+      const owner = new TeamOwner(myTeam, OWNER_MODIFIER.NOTIFY);
+      expect(owner.toString()).toContain(' (always notify)');
+      expect(owner.toString().endsWith(' (always notify)')).toBe(true);
```

Comment:
You can replace these two with
```javascript
expect(owner.toString()).toMatch(/ \(always notify\)$/);
```

Reply:
Whoops, didn't mean to have both in here (I think it was the result of a merge/rebase/stash or something). Regardless, I'll swap out for this matcher. Done.